Automattic/jetpack
# Adding the Password Checker package to the composer.json file.

Fixes #

#### Changes proposed in this Pull Request:

* Adds the automattic/jetpack-password-checker package to the composer.json file.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to the `projects/plugins/jetpack` directory.
* Delete the `vendor` directory and `composer.lock` file.
* `composer update`
*  Verify that the `jetpack-password-checker` package is in the `projects/plugins/jetpack/vendor/automattic` directory.
